Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -705,11 +711,36 @@ - (void)loadView
 #endif
 
 #if ENABLE(LINEAR_MEDIA_PLAYER)
-        _enterVideoFullscreenButton = [self _createButtonWithExtrinsicContentSize:buttonSize];
-        [_enterVideoFullscreenButton setConfiguration:cancelButtonConfiguration];
-        [_enterVideoFullscreenButton setImage:[[UIImage systemImageNamed:@"video"] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate] forState:UIControlStateNormal];
+        _enterVideoFullscreenButton = [UIButton buttonWithType:UIButtonTypeSystem];
+        [self _setupButton:_enterVideoFullscreenButton.get()];
+
+        UIButtonConfiguration *fullscreenButtonConfiguration = [UIButtonConfiguration filledButtonConfiguration];
+        fullscreenButtonConfiguration.imagePadding = 9;
+        fullscreenButtonConfiguration.contentInsets = NSDirectionalEdgeInsetsMake(12, 18, 12, 22);
+        fullscreenButtonConfiguration.titleLineBreakMode = NSLineBreakByClipping;
+
+        RetainPtr imageConfiguration = [[UIImageSymbolConfiguration configurationWithTextStyle:UIFontTextStyleBody] configurationByApplyingConfiguration:[UIImageSymbolConfiguration configurationWithWeight:UIImageSymbolWeightMedium]];
+
+        fullscreenButtonConfiguration.image = [[UIImage systemImageNamed:@"cube" withConfiguration:imageConfiguration.get()] imageWithRenderingMode:UIImageRenderingModeAlwaysTemplate];
+
+        RetainPtr descriptor = [UIFontDescriptor preferredFontDescriptorWithTextStyle:UIFontTextStyleBody];
+        descriptor = [descriptor fontDescriptorByAddingAttributes:@{
+            UIFontWeightTrait : [NSNumber numberWithDouble:UIFontWeightMedium]
+        }];
+        RetainPtr fullscreenTitleFont = [UIFont fontWithDescriptor:descriptor.get() size:0];
+        RetainPtr buttonTitle = adoptNS([[NSMutableAttributedString alloc] initWithString:WebCore::fullscreenControllerViewSpatial()]);
```

Comment:
I think WTF::String has an NSString operator?

Reply:
yes, hence why it works with initWithString: which takes a NSString

Reply:
Ignore me, I thought you were creating an NSMutableString and not an attributed string.